scripts/create-amazon-ecs-scratch

```diff
@@ -8,7 +8,8 @@
 # TODO, move the cert-inclusion process into this step
 
 mkdir amazon-ecs-scratch # include other directories that are needed here
+mkdir amazon-ecs-scratch/tmp
 cd amazon-ecs-scratch
 tar --owner=0 --group=0 --mtime="1970-01-01 00:00:00" -cv . | docker import - "amazon/amazon-ecs-scratch:make"
 cd ..
-rmdir amazon-ecs-scratch
+rm -rf amazon-ecs-scratch
```

Comment:
I would prefer `rmdir amazon-ecs-scratch/{tmp,}` or rmdir with two arguments to -rf